src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Your random ball:\n"
+                + "humber - "
+                + number
+                + "\n"
+                + "color - "
+                + color
+                + "\n"
+                + "\n"
+                .trim();
```

Comment:
use System.lineSeparator() instead of '\n', and i think .trim() is redundant here
```suggestion
    public String toString() {
        return "Your random ball:\n"
                + "humber - "
                + number
                + "\n"
                + "color - "
                + color
                + "\n"
                + "\n"
                .trim();
```